src/plugins/data/common/search/aggs/buckets/_interval_options.ts

```diff
@@ -20,12 +20,15 @@
 import { i18n } from '@kbn/i18n';
 import { IBucketAggConfig } from './bucket_agg_type';
 
+export const autoInterval = 'auto';
+export const isAutoInterval = (value: unknown) => Boolean(value === autoInterval);
```

Comment:
`===` expression already returns a boolean value, there is no need in `Boolean()`